templates/docs/examples/index.html

```diff
@@ -2,6 +2,7 @@
 
 {% block custom_head %}
     <link rel="stylesheet" href="/static/build/css/build.css" />
+    <link rel="stylesheet" href="/static/build/css/docs/cookie-policy.css" />
```

Comment:
Thanks @anthonydillon.
We have cookie-policy styles included already in site.css, so I guess example templates could use that for consistency with other layouts.

Reply:
```suggestion
```

Reply:
I was worried something in the site.css could lead into the examples. The other option is to remove the cookie policy from examples and just have it on the site and docs?

Reply:
The template name may be misleading.
This is a template for examples listing page (and standalone examples listing), not examples themselves.

Examples themselves use their own layout template, they don't use docs/site CSS or cookie policy.